components/Toast.js

```diff
@@ -20,7 +36,7 @@ const StyledToast = styled.div`
   justify-content: space-between;
   align-items: stretch;
   padding: 24px;
-  background: ${props => (props.variantType ? props.variantType.bg : 'white')};
+  background: ${props => props.variantType.bg};
```

Comment:
Is this line still needed, since `bg` is defined in `variants`?

Reply:
yes, it's not needed. modified the code now which will only be working on `styled-system/variants`.